python/docs/pyspark.sql.rst

```diff
@@ -7,6 +7,7 @@ Module Context
 .. automodule:: pyspark.sql
     :members:
     :undoc-members:
+    :inherited-members:
```

Comment:
I checked every API to see if it affects API documentation. It only affects `Row` which inherits `tuple`. It adds two APIs inherited from tuple where it seems legitimate.